### ISSUE TITLE
Stubs with Regexps are invalid Wiremock stubs

### ISSUE THREAD

**Issue by marcingrzejszczak** [Codearte/accurest#74]
For Groovy DSL

``` groovy
io.codearte.accurest.dsl.GroovyDsl.make {
                request {
                method """PUT"""
                url """/fraudcheck"""
                body("""
                    {
                    "clientPesel":"${value(client(regex('[0-9]{10}')), server('1234567890'))}",
                    "loanAmount":99999}
                """
                )
                headers {
                    header("""Content-Type""", """application/vnd.fraud.v1+json""")
                }

            }
            response {
                status 200
                body( """{
    "fraudCheckStatus": "${value(client('FRAUD'), server(regex('[A-Z]{5}')))}",
    "rejectionReason": "Amount too high"
}""")
                headers {
                     header('Content-Type': 'application/vnd.fraud.v1+json')
                    }
            }

}
```

and such Wiremock stub

``` json
{
    "request": {
        "method": "PUT",
        "headers": {
            "Content-Type": {
                "equalTo": "application/vnd.fraud.v1+json"
            }
        },
        "url": "/fraudcheck",
        "bodyPatterns": [
            {
                "matches": "{\"clientPesel\":\"[0-9]{10}\",\"loanAmount\":\"99999\"}"
            }
        ]
    },
    "response": {
        "status": 200,
        "headers": {
            "Content-Type": "application/vnd.fraud.v1+json"
        },
        "body": "{\"fraudCheckStatus\":\"FRAUD\",\"rejectionReason\":\"Amount too high\"}"
    }
}
```

Exception is thrown

```
org.springframework.web.client.HttpServerErrorException: 500 Illegal repetition {"clientPesel":"[0-9]{10}","loanAmount":"123.123"}
    at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:94)
    at org.springframework.web.client.RestTemplate.handleResponseError(RestTemplate.java:615)
    at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:573)
    at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:529)
    at org.springframework.web.client.RestTemplate.exchange(RestTemplate.java:447)
    at com.blogspot.toomuchcoding.frauddetection.LoanApplicationService.sendRequestToFraudDetectionService(LoanApplicationService.java:44)
    at com.blogspot.toomuchcoding.frauddetection.LoanApplicationService.loanApplication(LoanApplicationService.java:33)
    at com.blogspot.toomuchcoding.LoanApplicationServiceSpec.should be rejected due to abnormal loan amount(LoanApplicationServiceSpec.groovy:43)
```

**Comment by OlgaMaciaszek** [comment]
We face two issues here:
1. In order for this file to be processed correctly, the opening and closing curly braces in the `matches:` argument have to be escaped
2. The number (99999) is being transcribed as a String - it won't be matched the request from the client

The correct `bodyPatterns` section of the generated file should look this way:

``` groovy
 "bodyPatterns": [
            {
                "matches": "\\{\"clientPesel\":\"[0-9]{10}\",\"loanAmount\":99999\\}"
            }
        ]
```

Will work on fixing it now.

**Comment by OlgaMaciaszek** [comment]
The issue appears to have been already solved as per accurest-0.7.1.

Closing issue.
